src/activation/sigmoid.rs

```diff
@@ -0,0 +1,9 @@
+use super::Activation;
+use crate::value::Value;
+
+impl Activation {
+    pub fn sigmoid(&self, value: Value) -> Value {
+        let one = Value::new(1.0);
```

Comment:
There should already be an existing public method that does what you are doing here, so perhaps it's better to replace this logic with `Value::one()`. It's indirect but the overhead is small and dependent.

Reply:
ah okay, was trying to find something like this

Reply:
changed it to a one liner